synapse/lib/stormtypes.py

```diff
@@ -7816,6 +7817,7 @@ def __init__(self, runt, vdef, path=None):
         self.locls.update({
             'iden': self.valu.get('iden'),
             'parent': self.valu.get('parent'),
+            'children': [view.iden for view in self._reqView().children()],
```

Comment:
I think i'd like this to yield heavy view objects, which may mean that we should convert to a function to prevent creating a whole tree for each view. Lets have a quick goback